test/images/volumes-tester/nfs/run_nfs.sh

```diff
@@ -17,10 +17,22 @@
 function start()
 {
 
+    # accept "-G gid" option
+    while getopts "G:" opt; do
```

Comment:
I would sleep better if you clear `gid` before parsing command line, just in case `-G` is not set and `$gid` has some value from parent shell.